scripts/chroot_reconst.sh

```diff
@@ -119,10 +117,6 @@ rm -f plymouth-theme-meilix-text_1.0-2_all.deb
 rm -f meilix-default-theme_1.0-2_all.deb
 rm -f systemlock_0.1-1_all.deb
 
-# Meilix Check Skript
-chmod +x meilix_check.sh
-./meilix_check.sh
-
```

Comment:
Comment this out if you want less verbosity. Don't delete.

Reply:
yea would be helpful in future for debugging if we required it by uncommenting it

Reply:
Ok, will do the same

Reply:
It would also be better if we comment out copying the meilix-check script into the chroot